### ISSUE TITLE
Static Content: DUB Logos

### ISSUE THREAD

**Issue by jayfo** [uwdub/web-dub#16]
We should preserve the DUB logo content available here:

http://dub.washington.edu/logo/

This issue is focused on preserving those logos, not updating or changing them. That could happen later, but is not part of the current discussion.

I have created a simple directory and stub file that could be edited for this content:

https://github.com/uwdub/web-dub/tree/master/logos

https://github.com/uwdub/web-dub/edit/master/logos/index.md

**Comment by jayfo** [comment]
Simply moving the files over is easy enough, but how to link them is the thing to figure out.

This is completely distinct from updating the logo. That's not part of this discussion.

This should happen before launch, but not necessarily before public.

**Comment by jayfo** [comment]
If we create a basic page for this, it will be available as we start to develop page template and organize other static content.
